### PR TITLE
Fix safewalk disabling clipping on ledges despite being off

### PR DESCRIPTION
**Pull request by Vonr**

### CODE REVIEW

**Review by RemainingToast — COMMENTED** [review]
the event is only called while the module is enabled and this would mean it gets returned before it even 'turns on' safe walk

**Comment by Vonr** [comment]
yeah i found that out

File: src/main/kotlin/dev/toastmc/toastclient/impl/module/movement/SafeWalk.kt [hunk 1]
```diff
@@ -8,8 +8,8 @@ object SafeWalk : Module("SafeWalk", Category.MOVEMENT) {
 
     @Subscribe
     fun on(event: ClipAtLedgeEvent) {
-        if(mc.player == null && isEnabled()) return
-        event.clip = true
+        if (mc.player == null) return
+        event.clip = true;
```
Comment (RemainingToast) [inline src/main/kotlin/dev/toastmc/toastclient/impl/module/movement/SafeWalk.kt:12]: un necessary semi colon

Comment (Vonr) [reply src/main/kotlin/dev/toastmc/toastclient/impl/module/movement/SafeWalk.kt]: bruh

Comment (Vonr) [reply src/main/kotlin/dev/toastmc/toastclient/impl/module/movement/SafeWalk.kt]: also that null check was always formatted like that

Comment (Vonr) [reply src/main/kotlin/dev/toastmc/toastclient/impl/module/movement/SafeWalk.kt]: one of the reasons for the google java formatting i did

**Pull request merged**